documentation/book/ref-list-of-kafka-cluster-resources.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='ref-list-of-kafka-cluster-resources-{context}']
+= List of resources created as part of Kafka cluster
+
+The resources created by the Cluster Operator in the {ProductPlatformName} cluster will be the following:
+
+`_<cluster-name>_-kafka`:: StatefulSet which is in charge of managing the Kafka broker pods.
+`_<cluster-name>_-kafka-brokers`:: Service needed to have DNS resolve the Kafka broker pods IP addresses directly.
+`_<cluster-name>_-kafka-bootstrap`:: Service can be used as bootstrap servers for Kafka clients.
+`_<cluster-name>_-kafka-config`:: ConfigMap which contains the Kafka ancillary configuration and is mounted as a volume by the Kafka broker pods.
+`_<cluster-name>_-kafka-brokers`:: Secret with Kafka broker keys.
+`_<cluster-name>_-zookeeper`:: StatefulSet which is in charge of managing the Zookeeper node pods.
+`_<cluster-name>_-zookeeper-nodes`:: Service needed to have DNS resolve the Zookeeper pods IP addresses directly.
+`_<cluster-name>_-zookeeper-client`:: Service used by Kafka brokers to connect to Zookeeper nodes as clients.
+`_<cluster-name>_-zookeeper-config`:: ConfigMap which contains the Zookeeper ancillary configuration and is mounted as a volume by the Zookeeper node pods.
+`_<cluster-name>_-zookeeper-nodes`:: Secret with Zookeeper node keys.
+`_<cluster-name>_-entity-operator`:: Deployment with Topic and User Operators. (Optional).
+`_<cluster-name>_-entity-operator-config`:: Configmap with ancillary configuration for Topic and User Operators.
```

Comment:
Actually we'll have two different ConfigMaps, one for TO and the other for UO named : 

`_<cluster-name>_-entity-topic-operator-config`
`_<cluster-name>_-entity-user-operator-config`

because they are at specific operator level and not at higher EO level.